extras/audioDrivers/SDL/SoundSDL_Music.h

```diff
@@ -14,24 +14,40 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
-#ifndef SOUNDSDL_MUSIC_H_INCLUDED
-#define SOUNDSDL_MUSIC_H_INCLUDED
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+#ifndef extras_audioDrivers_SDL_SoundSDL_Music_h
+#define extras_audioDrivers_SDL_SoundSDL_Music_h
+
+#include <driver/SoundHandle.h>
 
-#include "driver/SoundHandle.h"
 #include <SDL_mixer.h>
+
 #include <utility>
 #include <vector>
 
 class SoundSDL_Music : public SoundDesc
 {
 public:
-    explicit SoundSDL_Music(Mix_Music* music) : SoundDesc(SD_MUSIC), music(music) {}
-    explicit SoundSDL_Music(std::vector<char> data) : SoundDesc(SD_MUSIC), music(nullptr), data(std::move(data)) {}
+    explicit
+    SoundSDL_Music(Mix_Music* music)
+        : SoundDesc(SD_MUSIC)
+        , music(music)
+    {}
+    
+    explicit 
+    SoundSDL_Music(std::vector<char> data)
+        : SoundDesc(SD_MUSIC)
+        , music(nullptr)
+        , data(std::move(data))
+    {}
 
     void setInvalid() { isValid_ = false; }
-    /// Handle to the sound. Managed and freed by the driver
-    Mix_Music* music; /// Das Music-Handle.
+
+    Mix_Music* music; ///< Handle to the sound. Managed and freed by the driver
```

Comment:
I'd rather have them above the decl. Long lines make reading this harder. the  tripple-slash already marks it as doc

Reply:
I could move it back above if you want to. for variables its sometimes easier if its behind the name